bedrock/firefox/templates/firefox/windows-64-bit.html

```diff
@@ -0,0 +1,76 @@
+ {% endblock %}
+
+ {% block page_css %}
+   {{ css_bundle('windows-64-bit') }}
+ {% endblock %}
+
+ {% block content %}
+   <div class="mzp-c-hero">
+     <div class="hero-content">
+       <p>{{ _('64-bit') }}</p>
+       <h1>{{ _('A more secure Firefox.') }}</h1>
+       <img src="{{ static('protocol/img/logos/firefox/firefox.png') }}" width="50" height="50" alt="">
+     </div>
+   </div>
+   <div class="mzp-l-content">
+   <article class="mzp-c-article">
+    <p class="mzp-c-article-intro">
+      {% trans crashes='https://blog.mozilla.org/firefox/defeat-browser-crashes/' %}
+      Users on 64-bit Windows who download Firefox can get our 64-bit version by default. That means you get a more secure version of Firefox, one that also <a href="{{ crashes }}">crashes a whole lot less</a>. How much less? In our tests so far, 64-bit Firefox reduced crashes by 39% on machines with 4GB of RAM or more.
+      {% endtrans %}
+    </p>
+
+    <hr>
+    <div class="article-section">
+      <h2>{{ _('What’s the difference between 32-bit and 64-bit?') }}</h2>
+      <p>
+        {% trans ASLR='https://en.wikipedia.org/wiki/Address_space_layout_randomization' %}
+        Here’s the key thing to know: 64-bit applications can access more memory and are less likely to crash than 32-bit applications. Also, with the jump from 32 to 64 bits, a security feature called <a href={{ ASLR }}>Address Space Layout Randomization (ASLR)</a> works better to protect you from attackers. Linux and macOS users, fret not, you already enjoy a Firefox that’s optimized for 64-bit.
+        {% endtrans %}
+      </p>
+    </div>
+
+    <div class="article-section">
+      <h2>{{ _('How do you get 64-bit Firefox?') }}</h2>
+      <p>
+      {% trans version='https://support.microsoft.com/help/13443/windows-which-operating-system', check='https://support.mozilla.org/kb/update-firefox-latest-version' %}
+       If you’re running 64-bit Windows (<a href="{{ version }}">here’s how to check</a>), your Firefox may already be 64-bit. <a href="{{ check }}">Check your Firefox version</a> (in the “About Firefox” window) and look for “(32-bit)” or “(64-bit)” after the version number:
+      {% endtrans %}
+       <ul>
+         <li>{{ _('If you see “(32-bit)” and you are running Firefox 56.0 or older, updating to the latest Firefox version should automatically upgrade you to 64-bit.') }}</li>
+         <li>{{ _('If you see “(32-bit)” and are running Firefox 56.0.1 or newer, then your computer may not meet the minimum memory requirement for 64-bit (3 GB RAM or more). You can still manually install 64-bit Firefox, if you choose.') }}</li>
+       </ul>
+       {% trans %}
+       If you need to run 32-bit Firefox or manually install 64-bit Firefox, you can simply download and re-run the Windows (32-bit or 64-bit) Firefox installer from the
+       {% endtrans %}
+       <a href="{{ url('firefox.all') }}">{{_('Firefox platforms and languages download') }} page.</a>
```

Comment:
This link text and the word `page` at the end should be part of the `trans` block above, as they are part of that same sentance?

Reply:
I agree, but was unsure how to pass the URL helper as a variable to the link.
`{% trans all='url('firefox.all')' %}` is unhappy.

Reply:
Try it without the quotes `{% trans all=url('firefox.all') %}`